ssl-config-core/src/main/scala/com/typesafe/sslconfig/ssl/FakeSSLTools.scala

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) 2015 - 2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package com.typesafe.sslconfig.ssl
```

Comment:
Should we move it to a `testkit` package to make it clear that it should not be used in production code?

Reply:
Maybe one day, for now this is kind of like `FakeKeyStore`. Not really for production, but not dangerous like a `lazy val` is to have here.

Reply:
I don't get your point about `lazy val`.

Reply:
@renatocaval not as dangerous as https://github.com/lightbend/ssl-config/pull/119#discussion_r221280971